### ISSUE TITLE
Mixed Reality Feature Tool requires outdated .NET version

### ISSUE THREAD

**Issue by codeling** [MicrosoftDocs/mixed-reality#637]
The Mixed Reality Feature Tool provided on this page (which as far as my web search went, is the newest version available?) requires version 5.0 of the .NET framework, for which the Microsoft pages say:

> This release has reached end of life, meaning it is no longer supported. We recommend moving to a supported release, such as [6.0 Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime). See our [support policy](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) for more details.

Above Quote is from https://dotnet.microsoft.com/en-us/download/dotnet/5.0/runtime?cid=getdotnetcore, where the MixedRealityFeatureTool.exe takes one after running it, after pressing Yes on the message
```
---------------------------
MixedRealityFeatureTool.exe
---------------------------
To run this application, you must install .NET.

Would you like to download it now?
---------------------------
Yes   No   
---------------------------
```

Is this tool not being supported properly anymore?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3a0405e5-aef5-5ed3-d9a1-960b4065300b
* Version Independent ID: dd9cd8d6-caea-6146-fd71-c5b3605435c4
* Content: [Welcome to the Mixed Reality Feature Tool - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/welcome-to-mr-feature-tool)
* Content Source: [mixed-reality-docs/mr-dev-docs/develop/unity/welcome-to-mr-feature-tool.md](https://github.com/MicrosoftDocs/mixed-reality/blob/docs/mixed-reality-docs/mr-dev-docs/develop/unity/welcome-to-mr-feature-tool.md)
* Product: **mixed-reality**
* Technology: **mr-dev**
* GitHub Login: @Sean-Kerawala
* Microsoft Alias: **sekerawa**

**Comment by laultman** [comment]
I found this issue, same as mine #650.

**Comment by MaxWang-MS** [comment]
This issue has been fixed in 1.0.2209.0. Please download the new version at https://aka.ms/mrfeaturetool.
